HISTORY.rst

```diff
@@ -2,6 +2,13 @@
 History
 =======
 
+v0.7.2 (TBA)
+------------
+
+* Cover all possible exception cases dealt by ``extruct()`` ``errors`` attribute for values ``strict``, ``log`` and ``ignore``
+* avoid including ``itemprop`` from child ``itemscope`` when using ``itemref``
+* proper processing order for ``itemref``
```

Comment:
Would you mind mentioning which extractors / semantic markup formats it affects?

Reply:
Sure @kmike. Done!